index.js

```diff
@@ -17,8 +16,17 @@ function renderPlayer(player) {
     <img src="${player.photo}" alt="${player.name}">
     <p class="likes">${player.likes} likes</p>
     <button class="like-button">❤️</button>
-  `
+    `
+  //DEVLIVERABLE #3 *****************************
+
+  const likeButton = playerDiv.querySelector("button.like-button")
+  likeButton.addEventListener("click", function(event) {
+    const likes = playerDiv.querySelector("p.likes")
+    const totalLikes = parseInt(likes.textContent)
+    likes.textContent = totalLikes + 1 + " likes"
+  })
```

Comment:
Nice, looks good! You can also access the player's likes directly, using the `player` variable:

```js
  const likeButton = playerDiv.querySelector("button.like-button")
  likeButton.addEventListener("click", function(event) {
    const likes = playerDiv.querySelector("p.likes")
    player.likes++
    likes.textContent = player.likes + " likes"
  })
```